### DISCUSSION TITLE
GitHub Pages and plugins

### DISCUSSION THREAD

**Discussion by drewgrif** [squidfunk/mkdocs-material#7482] [Q&A]
I am going to ask a dumb question. 

I want to be able to use the plugins that I am installing on my local machine on my GitHub Pages site.  Is this even possible?

**Comment by kamilkrzyskow** [comment]
Hello @drewgrif,
the GitHub workflow runner is simply another computer executing terminal commands. 
To use the plugins in your GitHub Actions you just need to install them the same way as you would on your local machine.

Similarly to the `pip install mkdocs-material` you can add other `pip install ...` commands, or add another module after the space `pip install name1 name2`
https://squidfunk.github.io/mkdocs-material/publishing-your-site/#__codelineno-0-28

Another approach would be to use a `requirements.txt` file together with `pip install -r requirements.txt`
